src/frontend/components/UI/ToggleSwitch/index.tsx

```diff
@@ -6,7 +6,7 @@ import './index.css'
 interface Props {
   htmlId: string
   handleChange: ChangeEventHandler<HTMLInputElement>
-  value: boolean
+  value: unknown
```

Comment:
Only improves convenience a bit. For EGL-Sync, we store a string value that dictates if the switch is toggled, so instead of passing `value={Boolean(eglPath)}`, you can just pass `value={eglPath}` directly (and it takes care of converting it to a Boolean)

We probably also have other uses of this component where the same could be applied, I haven't really checked (and even then, that might be better in a separate PR)